test/blackbox-tests/test-cases/re-exported-deps.t/run.t

```diff
@@ -17,29 +17,29 @@ transtive deps expressed in the dune-package
   (files
    (lib
     (META
-     dune-package
-     ccc/ccc.ml
-     ccc/ccc.cmi
-     ccc/ccc.cmx
-     ccc/ccc.cmt
-     ccc/ccc.cma
-     ccc/ccc.cmxa
-     ccc/ccc$ext_lib
-     bbb/bbb.ml
-     bbb/bbb.cmi
-     bbb/bbb.cmx
-     bbb/bbb.cmt
-     bbb/bbb.cma
-     bbb/bbb.cmxa
-     bbb/bbb$ext_lib
-     aaa/aaa.ml
+     aaa/aaa$ext_lib
+     aaa/aaa.cma
      aaa/aaa.cmi
-     aaa/aaa.cmx
      aaa/aaa.cmt
-     aaa/aaa.cma
+     aaa/aaa.cmx
      aaa/aaa.cmxa
-     aaa/aaa$ext_lib))
-   (libexec (ccc/ccc.cmxs bbb/bbb.cmxs aaa/aaa.cmxs)))
+     aaa/aaa.ml
+     bbb/bbb$ext_lib
+     bbb/bbb.cma
+     bbb/bbb.cmi
+     bbb/bbb.cmt
+     bbb/bbb.cmx
+     bbb/bbb.cmxa
+     bbb/bbb.ml
+     ccc/ccc$ext_lib
+     ccc/ccc.cma
+     ccc/ccc.cmi
+     ccc/ccc.cmt
+     ccc/ccc.cmx
+     ccc/ccc.cmxa
+     ccc/ccc.ml
+     dune-package))
+   (libexec (aaa/aaa.cmxs bbb/bbb.cmxs ccc/ccc.cmxs)))
```

Comment:
Looks like the quality of outputs generally improved across the tests: the output is a lot more sorted, which probably means it's more deterministic.

Reply:
Yeah, my changes modified the output, so I added a sort so that it's the last time we re-order things.

Reply:
Ah, I see, I actually missed the new sorting.